clients/go/dkv/client.go

```diff
@@ -0,0 +1,248 @@
+package dkv
+
+import (
+	"fmt"
+	"github.com/OneOfOne/xxhash"
+	"github.com/dgraph-io/ristretto"
+	"github.com/flipkart-incubator/dkv/pkg/ctl"
+	"github.com/flipkart-incubator/dkv/pkg/serverpb"
+
+	"log"
+)
+
+type DKVServerRole string
```

Comment:
Please include GoDoc. Else, check if this needs to be exposed.

Reply:
Done